andela/ah-shakas
# [CHORE #160577473] - Document API using swagger

### What does this PR do?

- Documents API using swagger
### Description of Task to be completed?
- [ah-shakas](https://github.com/andela/ah-shakas) has been documented using swagger
### How should this be manually tested?
- Clone [ah-shakas](https://github.com/andela/ah-shakas) repo `$ git clone https://github.com/andela/ah-shakas.git`
- export db credentials. please note that there is no spaces before and after `=`

```
      export DB_NAME="name of your database"
      export DB_PASSWORD="passord to your database user"
      export DB_HOST="your local server eg (127.0.0.1)"
      export DB_USER="your database username eg (postgres)"
```
_Note- if you have no password set for your database user leave the password field blank i.e `export DB_PASSWORD=""`_
- On the project's root directory run `python manage.py runserver`
- visit http://127.0.0.1:8000/
- Done! 🙂
### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/160577473
